src/ray/raylet/scheduling_resources.h

```diff
@@ -99,6 +99,11 @@ class ResourceSet {
   ///         False otherwise.
   bool GetResource(const std::string &resource_name, double *value) const;
 
+  /// Return the number of CPUs.
+  ///
+  /// \return Number of CPUs.
+  double GetNumCPUs() const;
```

Comment:
I think the convention we've been using is to lower-case acronyms, so it should be `GetNumCpus`.